junit-jupiter-api/src/main/java/org/junit/jupiter/api/Assertions.java

```diff
@@ -870,6 +871,51 @@ public static void assertIterableEquals(Iterable<?> expected, Iterable<?> actual
 		AssertIterableEquals.assertIterableEquals(expected, actual, messageSupplier);
 	}
 
+	// --- assertLinesMatch ----------------------------------------------------
+
+	/**
+	 * <em>Asserts</em> that {@code expected} list of {@linkplain String}s matches {@code actual}
+	 * list.
+	 *
+	 * <p>This method differs from other assertions that effectively only check
+	 * {@link String#equals(Object)} by using the following staged match-making algorithm:
```

Comment:
Sorry @sormuras, I realise that I was unclear; I thought that only the comma on line 880 needed to be removed, not also the one on line 881.

Hope about we change the middle of the sentence so it's clearer - something like:
> This method differs from other assertions that effectively only check {@code ...}, in that it uses the following staged match-making algorithm: